src/risedevtool/src/task/compute_node_service.rs

```diff
@@ -61,11 +61,7 @@ impl ComputeNodeService {
             .arg("--parallelism")
             .arg(&config.parallelism.to_string())
             .arg("--total-memory-bytes")
-            .arg(&config.total_memory_bytes.to_string())
-            .arg("--memory-control-policy")
-            .arg(&config.memory_control_policy)
-            .arg("--streaming-memory-proportion")
```

Comment:
Can you help to confirm these arguments are not used in cloud yet? @arkbriar

Reply:
AFAIK, neither the operator nor the cloud control plane use them.